File: AndroidApp/app/src/main/java/woutergijsbert/md2/avans/nl/restaurantinfo/models/Address.java
package woutergijsbert.md2.avans.nl.restaurantinfo.models;

public class Address {

    public String street;
    public String zipcode;
    public String city;
    public String region;
    public String country;

    public Address(String street, String zipcode, String city, String region, String country) {
        this.street = street;
        this.zipcode = zipcode;
        this.city = city;
        this.region = region;
        this.country = country;
    }
}

File: NativPokemon_gw/NativPokemon_gw/Address.swift
//
//  Address.swift
//  NativPokemon_gw
//
//  Created by Wouter van Geel on 09/04/16.
//  Copyright © 2016 gijsbertvancorstanje_woutervangeel. All rights reserved.
//

import Foundation

class Address {
    let street: String
    let zipcode: String
    let city: String
    let region: String
    let country: String
    
    init(_street: String, _zipcode: String, _city: String, _region: String, _country: String){
        
        street = _street
        zipcode = _zipcode
        city = _city
        region = _region
        country = _country
    }
}